miscellaneous-commands/userconfig.md

```diff
@@ -1,6 +1,6 @@
 # userconfig
 
-<TODO: describe>
+This command lets you configure some personal settings regarding the types of notifications you get from MoonlightBot.
```

Comment:
notifications aren't the only thing here (think of locale), i'm planning to add more options to this command as i develop things